L/Libtask/build_tarballs.jl

```diff
@@ -1,52 +1,58 @@
 # Note that this script can accept some limited command-line arguments, run
 # `julia build_tarballs.jl --help` to see a usage message.
-using Pkg
-using BinaryBuilder
+using BinaryBuilder, Pkg
 
-name = "Libtask"
-version = v"0.3.2"
-commit_id = "fbe338053f402d76524d0a01f3796dd7da90b781"
+julia_version = v"1.3.1"
 
-# see https://github.com/JuliaPackaging/BinaryBuilder.jl/issues/336
-# ENV["CI_COMMIT_TAG"] = ENV["TRAVIS_TAG"] = "v" * string(version)
+name = "Libtask"
```

Comment:
BTW I don't see a good reason for renaming this to `Libtask_julia` or so -- why should it matter that this links against libjulia_jll? We don't generally mention other dependencies in the name, do we?

Reply:
I got the idea from the name of the library `libcxxwrap_julia` but the main argument would be that, contrary to what the name suggests, this is not a build file for a binary package of the C library [Libtask](https://swtch.com/libtask/) but rather an implementation of [task copying](https://github.com/JuliaLang/julia/issues/4085) for Julia (which is mainly/only used by particle filter implementations in Turing).